client/src/menu/Anchor.tsx

```diff
@@ -270,3 +302,31 @@ function ErrorCard(props: { error: Error, onClose: () => void }) {
         <div>{props.error.body}</div>
     </div>
 }
+
+
+
+
+export function chatMessageToAudio(msg: ChatMessage): string | null {

```

Comment:
Make an type for audio file
e.g.
```tsx
type AudioFile = `audio/${string}.mp3`
```
or
```tsx
type Audio = "audio/vine_boom.mp3" | "audio/swag.mp3" /* etc etc */
```
Or if you really want, just:
```tsx
type Audio = string
```
It would just be nice to have a type for these function signatures. Cause it doesn't just return a string -- that string has meaning as representing audio